### PR TITLE
feat(electron): Session Replay for Electron

### LINKED ISSUE #6201: Replay: Electron SDK docs
### Core or SDK?

Platform/SDK

### Which part? Which one?

Electron

### Description

Electron has support for Session Replay but we don't have documentation to point users to

Please also add it to the include that is displayed at the top of this page:
https://docs.sentry.io/platforms/javascript/session-replay/configuration/

### Suggested Solution

Similar to how we documented Capacitor: https://github.com/getsentry/sentry-docs/pull/6143

### PR DESCRIPTION
**Pull request by timfish**
Closes #6201

### CODE REVIEW

**Review by lizokm — APPROVED** [review]
LG from my end.

File: src/platform-includes/session-replay/pre-requisites/javascript.capacitor.mdx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+For the sentry-replay integration to work, you must have the [Sentry Capacitor SDK package](https://www.npmjs.com/package/@sentry/capacitor) installed. The minimum version required for the SDK is `0.11.1`.
```
Comment (bruno-garcia) [inline src/platform-includes/session-replay/pre-requisites/javascript.capacitor.mdx:1]:
```suggestion
For the sentry-replay integration to work, you must have the [Sentry Electron SDK package](https://www.npmjs.com/package/@sentry/electron) installed. The minimum version required for the SDK is `4.2.0`.
```

File: src/platform-includes/session-replay/pre-requisites/javascript.capacitor.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+If you're on an older version of the SDK, please check the [troubleshooting
+document to help on
+migration](https://docs.sentry.io/platforms/javascript/guides/capacitor/troubleshooting/).
```
Comment (bruno-garcia) [inline src/platform-includes/session-replay/pre-requisites/javascript.capacitor.mdx:4]:
```suggestion
migration](https://docs.sentry.io/platforms/javascript/guides/electron/troubleshooting/).
```

**Review by bruno-garcia — APPROVED** [review]
Thanks Tim!

**Pull request merged**